src/Sylius/Bundle/ShopBundle/templates/account/address_book/index/addresses/default/content/address.html.twig

```diff
@@ -0,0 +1,3 @@
+<div>
+    {% include '@SyliusShop/shared/address.html.twig' with {'address': hookable_metadata.context.default_address} %}
```

Comment:
Define this templates in Twig Hooks configuration instead of including it here

Reply:
Since address template has been reworked in #16710, I will implement this hook after this PR is merged.